scripts/render.py

```diff
@@ -255,7 +259,7 @@ class RenderTrajectory:
     """Path to config YAML file."""
     rendered_output_names: List[str] = field(default_factory=lambda: ["rgb"])
     """Name of the renderer outputs to use. rgb, depth, etc. concatenates them along y axis"""
-    traj: Literal["spiral", "filename"] = "spiral"
```

Comment:
Now that we are adding more options, can you add a description of each option in the class docstring.

Reply:
I tried expanding docstring a bit in https://github.com/nerfstudio-project/nerfstudio/pull/1508/commits/cf01d605a1b67f6ea435d8cac37ff8a07f9376c9, do you think it is good or a bit verbose?

Reply:
Sorry, I should have been more specific. I was thinking the main docstring could be updated to something like,

```
"""Load a checkpoint, render a trajectory, and save to a video file.
The following trajectory options are available,
filename: Load from trajectory created using viewer or blender vfx plugin.
interpolate: Create trajectory by interpolating between eval dataset images.
spiral: Create a spiral trajectory (can be hit or miss).
"""
```

Reply:
Ah I see! Sorry for the confusion. :)

I've added your suggestions here b8f7eb8a17ea35abec1c422db109d1eb3f088e2e.